common-methods.go

```diff
@@ -114,7 +114,15 @@ func newClientFromAlias(alias string, urlStr string) (Client, *probe.Error) {
 	// credentials from the match found in the config file.
 	s3Config := new(Config)
 	s3Config.AccessKey = hostCfg.AccessKey
-	s3Config.SecretKey = hostCfg.SecretKey
+
+	// secretKey retrieved from the environement overrides the one
+	// present in the config file
+	if secretKeyEnv := os.Getenv("MC_SECRET_" + s3Config.AccessKey); isValidSecretKey(secretKeyEnv) {
```

Comment:
We can perhaps throw a warning message, if 'isValidSecretKey' fails, we are falling back to the config file.